ngraph/src/ngraph/pass/constant_folding_slice.cpp

```diff
@@ -71,46 +74,8 @@ void pass::ConstantFolding::construct_constant_slice()
         case element::Type_t::u1:
             NGRAPH_CHECK(false, "Encountered 'u1' element type in fold_constant_slice");
             break;
```

Comment:
Please add default case to avoid compilation error on macOS.

Reply:
done